tests/features/demo/demo_june.feature

```diff
@@ -0,0 +1,202 @@
+@api
+Feature: June 2016 demo
+  As users of the website
+  I should be able to interact with the website and manage content.
+
+  Scenario: Manage collection and view collection scenarios.
+    Given collections:
+      | title          | description                           | logo     | moderation |
+      | S.H.I.E.L.D.   | Well, they are mostly flying around.  | logo.png | yes        |
+      | x-Men          | Based on Proffessor Xavier's mantion. | logo.png | no         |
+      | Avengers       | Based on Tony stark's tower.          | logo.png | yes        |
+      | Fantastic four | Based on Reed Richard's tower.        | logo.png | yes        |
+    And solutions:
+      | title                     | description                                | documentation | moderation | collection   |
+      | Avengers initiative       | Gather the strongest into a group.         | text.pdf      | no         | S.H.I.E.L.D. |
+      | Project Tahiti            | Rejuvenate deadly wounds and erase memory. | text.pdf      | yes        | S.H.I.E.L.D. |
+      | Hellicarrier              | Provide a flying fortress as headquarters. | text.pdf      | no         | S.H.I.E.L.D. |
+      | Project 'Captain America' | Bring 'Captain america' back into action.  | text.pdf      | yes        | S.H.I.E.L.D. |
+    And users:
+      | name      | pass                 | mail                    | roles     |
+      | Stan lee  | cameoineverymovie    | stan.lee@example.com    | moderator |
+      | Nick Fury | ihaveasecret         | nick.fury@example.com   |           |
+      | Wolverine | smellslikemetalclaws | logan.x.men@example.com |           |
+    And user memberships:
+      | collection   | user      | roles                              |
+      | S.H.I.E.L.D. | Nick Fury | administrator, facilitator, member |
+      | x-Men        | Wolverine | facilitator, member                |
+      | Avengers     | Wolverine | member                             |
+    And news content:
+      | title                       | field_news_kicker                       | body                                                                                                                                |
+      | Phil Coulson is down        | Phil Coulson fell by the hands of Loki. | Phil Coulson tried to stop Loki from escaping and was killed by him.                                                                |
+      | Phoenix is down             | Wolverine took down Jean Gray.          | In an epic battle, Wolverine had to give the final blow to his great love, Jean Gray as she lost control to the Phoenix inside her. |
+      | S.H.I.E.L.D. is infiltrated | Winter soldier was spotted in action.   | As S.H.I.E.L.D. Hellicarrier is being taken down by the Winter soldier, we are also trying to spot the Hydra agents.                |
+    # The field_news_parent refers to a parent solution.
```

Comment:
OK good that you clarify it here. I'm guessing these steps are never intended to be executed by hand on an UAT environment right? Then it's fine to take some liberties with the 'business language' aspect of BDD.